### ISSUE TITLE
Press backspace key to delete rows or columns after select multiple empty rows or columns

### ISSUE THREAD

**Issue by Jocs** [marktext/marktext#1557]
### Description

- [x] Can you reproduce the issue? 

### Steps to reproduce

1. Create a table with multiple rows and columns.
2. Select multiple columns like bellow.
<img width="404" alt="屏幕快照 2019-10-29 下午6 15 47" src="https://user-images.githubusercontent.com/9712830/67758179-26791600-fa78-11e9-8533-1c0265be2ef9.png">

3. Press `backspace`

**Expected behavior:**

Selected two columns are removed.

**Actual behavior:**

Noting happened.

**Link to an example: [optional]**

### Versions

- Mark Text: develop
- Operating system: all

**Comment by adrienbeau** [comment]
I think the proposed "backspace" shortcut should delete the text inside each selected table cell rather than removing the columns. This is what most spreadsheet programs do.

**Comment by Jocs** [comment]
>I think the proposed "backspace" shortcut should delete the text inside each selected table cell rather than removing the columns. This is what most spreadsheet programs do.

Yes, remove all the contents at first press `backspace` and remove columns or rows at the second time when they are all empty cells.

**Comment by Jocs** [comment]
Thank you for the report! This has been fully addressed in the new editor engine (@muyajs/core). Pressing Backspace or Delete over a frozen multi-cell table selection now works in two stages: the first press clears the content of all selected cells (keeping the selection frozen), and a second press on now-empty whole-column or whole-row selections removes those columns or rows from the table. This matches the behavior described in the discussion. Please try it in the latest release.
